test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Models/PackageItemListViewModelTests.cs

```diff
@@ -37,5 +58,401 @@ public void LocalSources_PackageReader_NotNull()
             PackageReaderBase reader = func();
             Assert.IsType(typeof(PackageArchiveReader), reader);
         }
+
+        [Fact]
+        public void IconUrl_WithMalformedUrlScheme_ReturnsDefault()
+        {
+            var iconUrl = new Uri("httphttphttp://fake.com/image.png");
+
+            var packageItemListViewModel = new PackageItemListViewModel()
+            {
+                IconUrl = iconUrl
+            };
+
+            BitmapSource result = GetFinalIconBitmap(packageItemListViewModel).Result;
+
+            VerifyImageResult(result);
+            Assert.Same(DefaultPackageIcon, result);
+        }
+
+        [Fact]
+        public void IconUrl_WhenFileNotFound_ReturnsDefault()
+        {
+            var iconUrl = new Uri(@"C:\path\to\image.png");
+
+            var packageItemListViewModel = new PackageItemListViewModel()
+            {
+                IconUrl = iconUrl
+            };
+
+            BitmapSource result = GetFinalIconBitmap(packageItemListViewModel).Result;
+
+            VerifyImageResult(result);
+            Assert.Same(DefaultPackageIcon, result);
+        }
+
+        [Fact]
+        public void IconUrl_RelativeUri_ReturnsDefault()
+        {
+            // relative URIs are not supported in viewmodel.
+            var iconUrl = new Uri(@"resources/testpackageicon.png", UriKind.Relative);
+            var packageItemListViewModel = new PackageItemListViewModel()
+            {
+                IconUrl = iconUrl
+            };
+
+            BitmapSource result = GetFinalIconBitmap(packageItemListViewModel).Result;
+
+            VerifyImageResult(result);
+            Assert.Same(DefaultPackageIcon, result);
+        }
+
+        [Fact]
+        public void IconUrl_WithLocalPathAndColorProfile_LoadsImage()
+        {
+            var iconUrl = new Uri(@"resources/grayicc.png", UriKind.Relative);
```

Comment:
`@` is unnecessary.

Reply:
fixed in next iteration...yes, fixed all the forward slash instances, left the backwards one.